causal_testing/specification/causal_dag.py

```diff
@@ -521,5 +521,14 @@ def identification(self, base_test_case: BaseTestCase):
         minimal_adjustment_set = min(minimal_adjustment_sets, key=len)
         return minimal_adjustment_set
 
+    def to_dot(self) -> str:
```

Comment:
Should this have a different name? 'to_dot' suggests you may be getting a `.dot` file back not a str?